.expeditor/templates/welcome.mustache

```diff
@@ -0,0 +1,7 @@
+Hello {{author}}! Thanks for the pull request!
+
+Here is what will happen next:
+1. Your PR will be reviewed by the maintainers.
+2. If everything looks good, one of them will approve it, and your PR will be merged.
```

Comment:
I know it's not changing anything from the existing message, but do we want to add anything about the revision process, or the "if not everything looks good" case?

Reply:
I'm open to suggestions 😁